systemtest/src/main/java/io/strimzi/systemtest/resources/operator/configuration/OlmConfiguration.java

```diff
@@ -109,7 +109,7 @@ public String getOlmAppBundlePrefix() {
     }
 
     public String getOlmOperatorDeploymentName() {
-        return olmAppBundlePrefix + "-v" + operatorVersion;
+        return Environment.OLM_OPERATOR_DEPLOYMENT_NAME + "-v" + operatorVersion;
```

Comment:
Wouldn't be better to introduce new atribute in OlmConfiguration as we have it with other configurations?